module/applications/global/prompt-factory.mjs

```diff
@@ -394,6 +408,38 @@ class ItemPromptFactory extends PromptFactory {
     } );
   }
 
+  async _chooseTierPrompt( ) {
+
+    const buttons = Object.entries( ED4E.tier ).map(
+      ( [ key, label ] ) => {
+        return {
+          action:  key,
+          label:   label,
+          icon:    "",
+          class:   `button-${ key }`,
+          default: false
+        };
+      }
+    );
+
+    const titleFlavor = game.i18n.format( "ED.Dialogs.Title.chooseTier", {
+      abilityName: this.document.name,
+    } );
+
+    return DialogClass.wait( {
+      rejectClose: false,
+      id:          "choose-tier-prompt",
+      uniqueId:    String( ++globalThis._appId ),
+      classes:     [ "earthdawn4e", "choose-tier-prompt", "flexcol" ],
+      window:      {
+        title:       titleFlavor,
```

Comment:
Again, why is this extracted as constant when not used anywhere else? It's not wrong, just preference when it comes to readability

Reply:
i do that for readability and debugbability but i think it can be added to the title itself